uwuno/src/main/java/com/learning/uwuno/controllers/playerController.java

```diff
@@ -0,0 +1,99 @@
+package com.learning.uwuno.controllers;
+import com.fasterxml.jackson.databind.JsonMappingException;
```

Comment:
The same thing applies here for jackson libraries.

Reply:
Maybe we should agree on a minimum number before using wild card? I personally thing 2/3 is ok? And is it normal to include self made packages by wild card as well? If it's normal we can prob just do that since the number of files is only going to increase and we'll have to go back and re-import. The spring stuff maybe we leave alone? I don't know how big those packages are.

Reply:
Yes, self-made for sure. The jackson ones will have namespace error.